### PR TITLE
[strings] Fix diplicated strings for shop types

### PR DESCRIPTION
**Pull request by strump**
Working on #5708 

In commit [#6a45a62](https://github.com/organicmaps/organicmaps/commit/6a45a621f22aff2f298bb83e2dda19cf492606e5) on the 25 Sep 2018 Arsentiy Milchakov created first draft of `string_types.txt` file. He added some duplicates at that time. E.g. in Ukrainian `shop-chocolate`, `shop-coffee`, `shop-music`, and `shop-musical_instrument` types got same translation.

One can run script to get all duplicates:

```shell
tools/python/strings_utils.py --print-duplicates
```

For shop types see output: https://gist.github.com/strump/eb87e31262921e27ba10dd58b0410536

I fixed Ukrainian, Czech, Danish, Elliniká/Greek, Basque, Finnish, Hungarian, Indonesian, Romanian, Slovak, Sweden, Turkish, Thai, Vietnamese, Korean, and Marathi translations. Not translated: Persian (`fa`), `zh-Hans`, `zh-Hant`.

### CODE REVIEW

File: android/app/src/main/res/values-el/strings.xml [hunk 1]
```diff
@@ -1708,7 +1708,7 @@
 	<string name="type.shop.stationery">Κατάστημα γραφικής ύλης</string>
 	<string name="type.shop.supermarket">Σούπερ μάρκετ</string>
 	<string name="type.shop.tattoo">Τατουάζ</string>
-	<string name="type.shop.tea">Κατάστημα</string>
+	<string name="type.shop.tea">Τσαγοπωλείο</string>
```
Comment (cryoranger) [inline android/app/src/main/res/values-el/strings.xml:1711]:
```suggestion
	<string name="type.shop.tea">Τεϊοπωλείο</string>
```

File: iphone/Maps/LocalizedStrings/el.lproj/Localizable.strings [hunk 2]
```diff
@@ -3551,7 +3551,7 @@
 
 "type.shop.tattoo" = "Τατουάζ";
 
-"type.shop.tea" = "Κατάστημα";
+"type.shop.tea" = "Τσαγοπωλείο";
```
Comment (cryoranger) [inline iphone/Maps/LocalizedStrings/el.lproj/Localizable.strings:3554]:
```suggestion
"type.shop.tea" = "Τεϊοπωλείο";
```

File: android/app/src/main/res/values-el/strings.xml [hunk 3]
```diff
@@ -1646,9 +1646,9 @@
 	<string name="type.shop.car_repair.tyres">Βουλκανιζατέρ</string>
 	<string name="type.shop.carpet">Χαλιά</string>
 	<string name="type.shop.chemist">Χημικά προϊόντα</string>
-	<string name="type.shop.chocolate">Κατάστημα</string>
+	<string name="type.shop.chocolate">Σοκολατοποιία</string>
 	<string name="type.shop.clothes">Κατάστημα ρούχων</string>
-	<string name="type.shop.coffee">Κατάστημα</string>
+	<string name="type.shop.coffee">Ψωνίστε με καφέ</string>
```
Comment (cryoranger) [inline android/app/src/main/res/values-el/strings.xml:1651]:
```suggestion
	<string name="type.shop.coffee">Καφεκοπτείο</string>
```

File: iphone/Maps/LocalizedStrings/el.lproj/Localizable.strings [hunk 4]
```diff
@@ -3427,11 +3427,11 @@
 
 "type.shop.chemist" = "Χημικά προϊόντα";
 
-"type.shop.chocolate" = "Κατάστημα";
+"type.shop.chocolate" = "Σοκολατοποιία";
 
 "type.shop.clothes" = "Κατάστημα ρούχων";
 
-"type.shop.coffee" = "Κατάστημα";
+"type.shop.coffee" = "Ψωνίστε με καφέ";
```
Comment (cryoranger) [inline iphone/Maps/LocalizedStrings/el.lproj/Localizable.strings:3434]:
```suggestion
"type.shop.coffee" = "Καφεκοπτείο";
```

**Comment by strump** [comment]
@cryoranger  thanks for clarification. I fixed translations according to your suggestions.

**Comment by biodranik** [comment]
@strump can you please squash all non-regenerated commits?

**Comment by strump** [comment]
@biodranik  Rebased. Now this PR has only 2 commits

File: android/app/src/main/res/values-ro/strings.xml [hunk 5]
```diff
@@ -1691,17 +1691,17 @@
...
 	<string name="type.shop.jewelry">Bijutier</string>
-	<string name="type.shop.kiosk">Magazin</string>
+	<string name="type.shop.kiosk">Chioșc</string>
 	<string name="type.shop.kitchen">Magazin de bucatarie</string>
 	<string name="type.shop.laundry">Spălătorie</string>
 	<string name="type.shop.mall">Mall</string>
 	<string name="type.shop.massage">Salon de masaj</string>
 	<string name="type.shop.mobile_phone">Magazin de telefoane mobile</string>
-	<string name="type.shop.money_lender">Magazin</string>
+	<string name="type.shop.money_lender">Creditor de bani</string>
```
Comment (vgrigoriu) [inline android/app/src/main/res/values-ro/strings.xml:1700]: This doesn't sound very idiomatic, but on the other hand it's hard to think of a better alternative. Google Maps calls this category "Agenție de facilitări credite".

Comment (vgrigoriu) [reply android/app/src/main/res/values-ro/strings.xml:1700]: Other options: "IFN" (non-bank financial institution), or simply "Credite".

Comment (strump) [reply android/app/src/main/res/values-ro/strings.xml]: Thanks. Replaced `Creditor de bani` with `Credite`

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**